tsconfig.json

```diff
@@ -18,7 +18,11 @@
     "lib": [
       "es2018",
       "dom"
-    ]
+    ],
+    "paths": {
+      "@genres/*": ["app/genres/*"],
```

Comment:
@k1k8r lets improve aliases for module? Will be better to make link to index file like this:

`"@genres/*": ["app/genres/*"],` => `"@genres": ["app/genres/index.ts"],`  or `@genres/interfaces": ["app/genres/interfaces/index.ts"],`